agent/build.gradle

```diff
@@ -30,6 +34,8 @@ dependencies {
 
 processResources {
     from configurations.mvnExtensions
+    from configurations.geMvnExtensionOnePrevious
+    from configurations.geMvnExtensionTwoPrevious
```

Comment:
This PR (not just this line) increase the complexity of managing the extension versions in this repository. I see two issues:
* I expect Dependabot to try to update these two lines to the most recent versions of Gradle Enterprise Maven Extension every time a new release occurs. We could either disable Dependabot for the agent and lost the benefits for all other dependencies, or ensure Dependabot PRs for these two lines are not merged. Both feel problematic.
* This just adds more places where we need to manually update version numbers on updates to the Gradle Enterprise Maven Extension that are not covered by Dependabot. See other references to versions present in this PR. I'd be interested in discussing a solution to this issue at some point.